htcanalyze/htcanalyze.py

```diff
@@ -758,8 +765,8 @@ def analyzed_summary(self, log_files: list_of_logs) -> log_inf_list:
 
             if occurred_errors:
                 create_file_list = list()
-                for i in range(len(occurred_errors["Event Number"])):
-                    create_file_list.append(file)
+                # TODO: make this a one-liner
+                create_file_list.extend([file for i in range(len(occurred_errors["Event Number"]))])
```

Comment:
Is this supposed to append the same `file` a number of times equal to the length of `occurred_errors['Event Number']`? Because that's what it seems to be doing. If so, then there's no need for a loop. You could just do: `occurred_errors['File'] = [file]*len(occurred_errors['Event Number'])`